### PR TITLE
UpdatedDisplayNames

### LINKED ISSUE #1105: Update display names of local gateway and wallet
Currently the generator names the local fabric wallet as local_fabric_wallet and gateway as local_fabric. These should be changed in vscode to be Local Fabric and Local Fabric Wallet

### PR DESCRIPTION
**Pull request by toggle007**
Edited the local_fabric & local_fabric_wallet display names.
closes #1105

Signed-off-by: Akshat Shah <akshatshah@akshats-mbp.hursley.uk.ibm.com>

### CODE REVIEW

File: client/cucumber/data/cucumber.code-workspace [hunk 1]
```diff
@@ -3,6 +3,30 @@
 		{
 			"path": "emptyProject",
 			"name": "emptyProject"
+		},
```
Comment (cazfletch) [inline client/cucumber/data/cucumber.code-workspace]: you shouldn't commit this change

File: client/src/commands/restartFabricRuntime.ts [hunk 2]
```diff
@@ -28,12 +29,16 @@ export async function restartFabricRuntime(): Promise<void> {
         title: 'IBM Blockchain Platform Extension',
         cancellable: false
     }, async (progress: vscode.Progress<{ message: string }>) => {
-        progress.report({ message: `Restarting Fabric runtime ${runtime.getName()}` });
+        let fabricRuntimeName: string = runtime.getName();
```
Comment (cazfletch) [inline client/src/commands/restartFabricRuntime.ts]: you don't need this if you can just use `FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME`

File: client/src/commands/restartFabricRuntime.ts [hunk 3]
```diff
@@ -28,12 +29,16 @@ export async function restartFabricRuntime(): Promise<void> {
...
+        if (fabricRuntimeName === FabricRuntimeUtil.LOCAL_FABRIC) {
+            fabricRuntimeName = FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME;
+        }
+        progress.report({ message: `Restarting Fabric runtime ${fabricRuntimeName}` });
 
         try {
             await runtime.restart(outputAdapter);
         } catch (error) {
-            outputAdapter.log(LogType.ERROR, `Failed to restart local_fabric: ${error.message}`, `Failed to restart local_fabric: ${error.toString()}`);
+            outputAdapter.log(LogType.ERROR, `Failed to restart Local Fabric: ${error.message}`, `Failed to restart Local Fabric: ${error.toString()}`);
```
Comment (cazfletch) [inline client/src/commands/restartFabricRuntime.ts]: This should use the variable

File: client/src/commands/startFabricRuntime.ts [hunk 4]
```diff
@@ -30,7 +31,11 @@ export async function startFabricRuntime(): Promise<void> {
         title: 'IBM Blockchain Platform Extension',
         cancellable: false
     }, async (progress: vscode.Progress<{ message: string }>) => {
-        progress.report({ message: `Starting Fabric runtime ${runtime.getName()}` });
+        let fabricRuntimeName: string = runtime.getName();
```
Comment (cazfletch) [inline client/src/commands/startFabricRuntime.ts]: again you don't need the if

File: client/src/commands/startFabricRuntime.ts [hunk 5]
```diff
@@ -39,7 +44,7 @@ export async function startFabricRuntime(): Promise<void> {
             await runtime.start(outputAdapter);
             await runtime.importWalletsAndIdentities();
         } catch (error) {
-            outputAdapter.log(LogType.ERROR, `Failed to start local_fabric: ${error.message}`, `Failed to start local_fabric: ${error.toString()}`);
+            outputAdapter.log(LogType.ERROR, `Failed to start Local Fabric: ${error.message}`, `Failed to start Local Fabric: ${error.toString()}`);
```
Comment (cazfletch) [inline client/src/commands/startFabricRuntime.ts]: you should use the variable

File: client/src/commands/stopFabricRuntime.ts [hunk 6]
```diff
@@ -31,7 +32,11 @@ export async function stopFabricRuntime(): Promise<void> {
         title: 'IBM Blockchain Platform Extension',
         cancellable: false
     }, async (progress: vscode.Progress<{ message: string }>) => {
-        progress.report({ message: `Stopping Fabric runtime ${runtime.getName()}` });
+        let fabricRuntimeName: string = runtime.getName();
```
Comment (cazfletch) [inline client/src/commands/stopFabricRuntime.ts]: same again

File: client/src/commands/stopFabricRuntime.ts [hunk 7]
```diff
@@ -41,7 +46,7 @@ export async function stopFabricRuntime(): Promise<void> {
         try {
             await runtime.stop(outputAdapter);
         } catch (error) {
-            outputAdapter.log(LogType.ERROR, `Failed to stop local_fabric: ${error.message}`, `Failed to stop local_fabric: ${error.toString()}`);
+            outputAdapter.log(LogType.ERROR, `Failed to stop Local Fabric: ${error.message}`, `Failed to stop Local Fabric: ${error.toString()}`);
```
Comment (cazfletch) [inline client/src/commands/stopFabricRuntime.ts]: same again

File: client/src/commands/teardownFabricRuntime.ts [hunk 8]
```diff
@@ -39,8 +40,11 @@ export async function teardownFabricRuntime(force: boolean = false): Promise<voi
         title: 'IBM Blockchain Platform Extension',
         cancellable: false
     }, async (progress: vscode.Progress<{ message: string }>) => {
-        progress.report({ message: `Tearing down Fabric runtime ${runtime.getName()}` });
-
+        let fabricRuntimeName: string = runtime.getName();
```
Comment (cazfletch) [inline client/src/commands/teardownFabricRuntime.ts]: same again

File: client/src/commands/teardownFabricRuntime.ts [hunk 9]
```diff
@@ -50,7 +54,7 @@ export async function teardownFabricRuntime(force: boolean = false): Promise<voi
             await runtime.teardown(outputAdapter);
             await runtime.deleteWalletsAndIdentities();
         } catch (error) {
-            outputAdapter.log(LogType.ERROR, `Failed to teardown local_fabric: ${error.message}`, `Failed to teardown local_fabric: ${error.toString()}`);
+            outputAdapter.log(LogType.ERROR, `Failed to teardown Local Fabric: ${error.message}`, `Failed to teardown Local Fabric: ${error.toString()}`);
```
Comment (cazfletch) [inline client/src/commands/teardownFabricRuntime.ts]: same again

File: client/src/commands/toggleFabricRuntimeDevMode.ts [hunk 10]
```diff
@@ -41,11 +42,15 @@ export async function toggleFabricRuntimeDevMode(): Promise<void> {
             title: 'IBM Blockchain Platform Extension',
             cancellable: false
         }, async (progress: vscode.Progress<{ message: string }>) => {
-            progress.report({ message: `Restarting Fabric runtime ${runtime.getName()}` });
+            let fabricRuntimeName: string = runtime.getName();
```
Comment (cazfletch) [inline client/src/commands/toggleFabricRuntimeDevMode.ts]: same again

File: client/src/commands/toggleFabricRuntimeDevMode.ts [hunk 11]
```diff
@@ -41,11 +42,15 @@ export async function toggleFabricRuntimeDevMode(): Promise<void> {
...
+            if (fabricRuntimeName === FabricRuntimeUtil.LOCAL_FABRIC) {
+                fabricRuntimeName = FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME;
+            }
+            progress.report({ message: `Restarting Fabric runtime ${fabricRuntimeName}` });
             try {
                 await runtime.restart(outputAdapter);
             } catch (error) {
-                outputAdapter.log(LogType.ERROR, `Failed to restart local_fabric: ${error.message}`, `Failed to restart local_fabric: ${error.toString()}`);
+                outputAdapter.log(LogType.ERROR, `Failed to restart Local Fabric: ${error.message}`, `Failed to restart Local Fabric: ${error.toString()}`);
```
Comment (cazfletch) [inline client/src/commands/toggleFabricRuntimeDevMode.ts]: same again

File: client/src/explorer/gatewayExplorer.ts [hunk 12]
```diff
@@ -177,9 +178,12 @@ export class BlockchainGatewayExplorerProvider implements BlockchainExplorerProv
                     arguments: [runtimeGateway]
                 };
 
+                let gatewayName: string;
+                gatewayName = FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME;
+
                 const treeItem: LocalGatewayTreeItem = await LocalGatewayTreeItem.newLocalGatewayTreeItem(
                     this,
-                    runtimeGateway.name,
+                    gatewayName,
```
Comment (cazfletch) [inline client/src/explorer/gatewayExplorer.ts]: you can just put the `FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME` here you don't need to create a variable

File: client/CHANGELOG.md [hunk 13]
```diff
@@ -135,7 +135,7 @@ We've reached 10,000 installs :tada: - a big thank you to everyone who has contr
 
 ## 0.3.2: March 14th 2019
 
-* Added "Import Package" command [#665](https://github.com/ibm-blockchain/blockchain-vscode-extension/issues/665)
+* Added "Import a Package" command [#665](https://github.com/ibm-blockchain/blockchain-vscode-extension/issues/665)
```
[the remark this answers is not in the document]
Comment (toggle007) [reply client/CHANGELOG.md]: Reverted (Will push all at once, along with the other changes)

**Pull request merged**